src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -271,6 +275,7 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
             actualDayParties: actualParties,
```

Comment:
according to the error on the ticket, it also need to cover the the hearingDate case.
when the hearingDate from actual date is not existing, clone it from the planned hearing date.